README.md

```diff
@@ -15,6 +15,10 @@ Access design documentation for features slotted in the 4.2 release of OpenShift
 - [Operators Overview](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/operators-overview/operators-overview)
 - [Taints and Tolerations](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/taints-tolerations/taints-tolerations)
 
+## OpenShift Container Storage
+- [OCS Install workflow](./web-console/Storage/OCS/OCS_Installation_Workflow.md)
```

Comment:
@shirimordechay Clicking the ocs install worklfow link is giving 404 error to me. Is there something I am doing wrong here?

Reply:
It's not merged yet, try this for now: https://github.com/shirimordechay/openshift-origin-design/blob/OCS-Install-workflow/web-console/Storage/OCS/OCS%20_Installation_Workflow.md